src/amo/actions/reviews.js

```diff
@@ -42,9 +42,15 @@ export const DELETE_ADDON_REVIEW: 'DELETE_ADDON_REVIEW' = 'DELETE_ADDON_REVIEW';
 export const UNLOAD_ADDON_REVIEWS: 'UNLOAD_ADDON_REVIEWS' =
   'UNLOAD_ADDON_REVIEWS';
 
+export type SimplifiedAddonType = {|
+  iconUrl?: string,
+  id: number,
+  name?: string,
+  slug: string,
+|};
+
 export type UserReviewType = {|
-  addonId: number,
-  addonSlug: string,
+  addon: SimplifiedAddonType,
```

Comment:
Even though Flow will catch mistakes, I agree that we should not call this `addon`. We should call it `reviewAddon` something.